<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>printer</class>
 <widget class="QWidget" name="printer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1,0">
     <item row="0" column="0" colspan="5">
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1,0,1,0,2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_w">
         <property name="text">
          <string>宽</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_w">
         <property name="text">
          <string>40</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_h">
         <property name="text">
          <string>高</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="lineEdit_h">
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_zt">
         <property name="text">
          <string>字体</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="comboBox_zt"/>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_zh">
         <property name="text">
          <string>字号</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QComboBox" name="comboBox_zh">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="5">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_wbpm">
         <property name="text">
          <string>外部品名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_wbkh">
         <property name="text">
          <string>外部款号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_ys">
         <property name="text">
          <string>颜色</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_cm">
         <property name="text">
          <string>尺码</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton_xb">
         <property name="text">
          <string>性别</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_zypl">
         <property name="text">
          <string>主营品类</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_gys">
         <property name="text">
          <string>供应商</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_lxhh">
         <property name="text">
          <string>乐销货号</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_id">
         <property name="text">
          <string>商品ID</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButton_barcode">
         <property name="text">
          <string>条形码 </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="QLabel" name="label_page">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_bc">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_zr">
       <property name="text">
        <string>载入</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_mr">
       <property name="text">
        <string>恢复默认标签 </string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_dqmb">
       <property name="text">
        <string>使用当前模板</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_qx">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
